### DISCUSSION TITLE
Wish to have internationalized language pack

### DISCUSSION THREAD

**Discussion by blinlincom** [Sketchware-Pro/Sketchware-Pro#1066] [Ideas]
Hopefully there will be new internationalized language packs to get more people using the software.

**Comment by danielclement921-tech** [comment]
How can I save my sketchware project to my phone

**Comment by webmturn** [comment]
I hope so too.

**Comment by danielclement921-tech** [comment]
Please how can I use unity ads in sketchware

**Comment by webmturn** [comment]
我解决了，查看我的分支
